examples/crystal.py

```diff
@@ -269,6 +271,9 @@ def simulate_crystal_with_gaussian_beam(
         (hx, hy, hz), (x, y, z))`. The fields are returned for an xz slice centered
         on the incident beam.
     """
+    wavelengths = jnp.expand_dims(jnp.atleast_1d(wavelengths), axis=(1, 2))
```

Comment:
We actually have a function in the utils module, `atleast_nd` which may be simpler to use here.

Reply:
Great! Can you choose we side to pad? I know numpy's `atleast_nd` prepends the array, when in this case we'd prefer to append.